internal/services/legacy/virtual_machine_unmanaged_disks_resource_test.go

```diff
@@ -282,7 +282,7 @@ func TestAccVirtualMachine_deleteVHDOptIn(t *testing.T) {
 func TestAccVirtualMachine_ChangeComputerName(t *testing.T) {
 	data := acceptance.BuildTestData(t, "azurerm_virtual_machine", "test")
 	r := VirtualMachineResource{}
-	data.ResourceTest(t, r, []acceptance.TestStep{
+	data.ResourceTestIgnoreRecreate(t, r, []acceptance.TestStep{
```

Comment:
As above, this test just cycles a `ForceNew` property, and doesn't provide any resource value since Terraform will always just delete and recreate it. We should remove the test.

Reply:
Removed